package.json

```diff
@@ -104,17 +104,18 @@
     "check-licenses": "wp-scripts check-licenses --production",
     "deploy": "grunt deploy",
     "dev": "wp-scripts start",
+    "diff-develop": "git diff develop --diff-filter=d --name-only",
     "env:start": "./bin/local-env/start.sh",
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "wp-scripts lint-style",
+    "lint:css": "npm run diff-develop | grep -E 'css$' | xargs wp-scripts lint-style",
     "lint:css:fix": "npm run lint:css -- --fix",
```

Comment:
Missing incorporation of the subset.

Reply:
Sure, is https://github.com/ampproject/amp-wp/pull/4441/commits/83e21556ba66a5a33464a9486e883c6ca990c785 what you had in mind by subset?